zeppelin-web/add_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+( sudo apt-get install libfontconfig -y || sudo yum install fontconfig -y ) || exit 0;
```

Comment:
@Leemoonsoo  I can adjust build order but this do not save enough time - its run about 40 minutes until fail. To be honest I don't understand why you so opposite to automatically install package which should be already installed for build? If you have newer version at your workstation package manager will not reinstall old package. This is a safe operation.